decapode/kafka_integration.py

```diff
@@ -0,0 +1,75 @@
+from typing import Awaitable, Optional
+import json
+import logging
+import requests
+
+from aiohttp import web
+from kafka import KafkaConsumer, KafkaProducer
+
+from decapode.config import KAFKA_URI, KAFKA_API_VERSION, DECAPODE_API_URL, DATAGOUV_API_URL
+from decapode.crawl import crawl_urls
+
+log = logging.getLogger("decapode-kafka")
+
+def produce(producer: KafkaProducer, id: str, data: dict) -> None:
+    key = id.encode("utf-8")
+
+    value = {
+        'service': 'udata',
+        'data': data,
+    }
+    producer.send(topic='decapode', value=value, key=key)
+    producer.flush()
+
+
+def get_latest_check(resource_id: str) -> Optional[int]:
+    '''Returns None if unsuccessful at fetching latest check, otherwise returns latest check status code.'''
+    resource_latest_check_url = f'{DECAPODE_API_URL}/api/checks/latest/?resource_id={resource_id}'
+    try:
+        return requests.get(resource_latest_check_url).json()['status']
+    except json.JSONDecodeError:
```

Comment:
We may want to catch other exception due to decapode service errors. Ex: catch ConnectionError, raise errors for requrest status.